build/yamls/antrea-octant.yml

```diff
@@ -0,0 +1,72 @@
+# yaml template for octant and antrea-octant-plugin
+# Right config parameters and Docker image must be specified.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: antrea-octant
+  namespace: kube-system
+  labels:
+    app: antrea
+    component: antrea-octant
+spec:
+  replicas: 1
+  strategy:
+    # Ensure the existing Pod is killed before the new one is created.
+    type: Recreate
+  selector:
+    matchLabels:
+      app: antrea
+      component: antrea-octant
+  template:
+    metadata:
+      labels:
+        app: antrea
+        component: antrea-octant
+    spec:
+      nodeSelector:
+        # Add the role to support running antrea-octant on specific node
+        # In this way, users can see UI via a fixed URL as http://$HOSTNAME:8900
+        role: antrea-octant
```

Comment:
Correct, it's not in the official doc of octant and it's only my steps when I deploy octant together with our plugin. Users can always change it if they want to run it in a random node. I am adding this because we usually want to see UI via a fixed URL and it is just simple to run it in a fixed node. I also added this in the doc.

Reply:
Thanks @salv-orlando .I have taken another thought of it and I think the comment really makes sense. Now I am thinking to add a NodePort type of service, so that antrea-octant can run in any node. Let me test it on the live testbed and update this part if it works.

Reply:
I did not add service definition to our sample because there are multiple ways to deploy such a service. Users may also encounter different problems when they choose different ways. But I do remove the taint node part and add a short description for exposing service after step 5.